packages/fxa-content-server/app/scripts/lib/auth-errors.js

```diff
@@ -157,6 +157,10 @@ var ERRORS = {
     errno: 140,
     message: t('Account already exists'),
   },
+  MAX_SECONDARY_EMAILS_REACHED: {
+    errno: 188,
+    message: t('You cannot have more than 3 secondary emails'),
```

Comment:
I would probably say something like `You have reached the maximum allowed secondary emails`, to keep it generic.